ledger/indexer-benchmark/src/main/scala/ledger/indexerbenchmark/IndexerBenchmark.scala

```diff
@@ -148,14 +148,10 @@ class IndexerBenchmark() {
              |  inputMapping.batchSize:     ${histogramToString(
             metrics.daml.parallelIndexer.inputMapping.batchSize.getSnapshot
           )}
-             |  inputMapping.duration:      ${histogramToString(
```

Comment:
We still have these metrics in the registry, albeit under different names. Why remove them?